client/src/components/App/Content/UsersManagement/UsersManagement.tsx

```diff
@@ -81,6 +82,38 @@ const UsersManagement: React.FC = () => {
             case UsersManagementTableHeadersNames.DESK: {
                 return row[cellName] ? row[cellName] : noDeskAssignment
             }
+            case UsersManagementTableHeadersNames.SOURCE_ORGANIZATION : {
+                return (
+                    <Select
+                        MenuProps={{
+                            anchorOrigin: {
+                                vertical: "bottom",
+                                horizontal: "left"
+                            },
+                            transformOrigin: {
+                                vertical: "top",
+                                horizontal: "left"
+                            },
+                            getContentAnchorEl: null
+                        }}
+                        variant='outlined'
+                        className={classes.sourceOrganization}
+                        label={sourceOrganizationLabel}
+                        value={row[cellName]}
```

Comment:
You know the value is the source organization. I think it's better to be explicit: `row.sourceOrganization` or what it's name is.

Reply:
fixed